packages/xo-web/src/xo-app/backup/overview/index.js

```diff
@@ -148,15 +148,21 @@ const SchedulePreviewBody = decorate([
           style={{ marginRight: '0.5em' }}
         />
         {job.runId !== undefined ? (
-          <ActionButton
-            btnStyle='danger'
-            handler={cancelJob}
-            handlerParam={job}
-            icon='cancel'
-            key='cancel'
-            size='small'
-            tooltip={_('formCancel')}
-          />
+          <Tooltip content={_('temporarilyDisabled')}>
+            <span>
+              <ActionButton
+                btnStyle='danger'
+                // TODO: enable the button once the issue with the job cancellation was fixed
```

Comment:
You can add the date of today, explain why we're disabling it and add the link to the issue.